tests/views/test_change_response_status.py

```diff
@@ -289,3 +296,45 @@ def test_respondent_name_not_in_metadata_for_completed_case_event(self, mock_req
         self.assertIn(b"221 BLOCKS", data)
         self.assertIn(b"Completed", data)
         self.assertNotIn(b"Jacky Turner", data)
+
+    @requests_mock.mock()
+    def test_not_started_status_is_present_for_completed_by_phone(self, mock_request):
+        mock_request.get(url_get_survey_by_short_name, json=survey)
+        mock_request.get(url_get_collection_exercises_by_survey, json=collection_exercise_list)
+        mock_request.get(url_get_business_by_ru_ref, json=business_reporting_unit)
+        mock_request.get(url_get_available_case_group_statuses, json=self.statuses)
+        mock_request.get(url_get_case_groups_by_business_party_id, json=case_groups_completed_by_phone)
+        mock_request.get(url_get_case_events, json=case_events)
+        mock_request.get(url_update_case_group_status, json=case_groups_not_started)
+        mock_request.get(url_get_case_by_case_group_id, json=[case])
+
+        response = self.client.get(f"/case/{ru_ref}/response-status?survey={short_name}&period={period}")
+
+        data = response.data
+        self.assertEqual(response.status_code, 200)
+        self.assertIn(b"19000001", data)
+        self.assertIn(b"Bolts and Ratchets", data)
+        self.assertIn(b"221 BLOCKS", data)
```

Comment:
Not major, but we really should start to assert against just what we are testing, so this can be removed. The whole test class really needs a refactor, but definitely not for this story